evoseal/examples/basic/basic_usage.py

```
@@ -9,7 +9,7 @@
 
 import json
 import os
-import subprocess
+import subprocess  # nosec - Not used in this file, but kept for future use
```

Flake8 (7.2.0)

[error] 'subprocess' imported but unused

(F401)